third_party/gpus/cuda_configure.bzl

```diff
@@ -1015,7 +1015,7 @@ def _create_dummy_repository(repository_ctx):
             "%{cuda_version}": _DEFAULT_CUDA_VERSION,
             "%{cudnn_version}": _DEFAULT_CUDNN_VERSION,
             "%{cuda_compute_capabilities}": ",".join([
-                "CudaVersion(\"%s\")" % c
+                "DeviceVersion::Parse(\"%s\")" % c
```

Comment:
We can restore the old behavior by making this `DeviceVersion::Parse(%s).ValueOrDie()`.  I admit I don't understand what this is doing or how to test it, though.  We'll need someone from the build team to look at this.

\summon @gunan